Test/VsVimSharedTest/CodeHygieneTest.cs

```diff
@@ -57,6 +57,7 @@ public void CodeNamespace()
         /// the Vim.Core assembly and not an actual reference. Too many ways that VS ships the DLL that
         /// it makes referencing it too difficult. Embedding is much more reliably.
         /// </summary>
+        // Manually verified for 2.10.0.6

```

Comment:
This bug is still present in the latest 17.3 release. I enabled the test locally, used the 17.4 preview toolset and verified thihs passed.